test/import/import-crd.test.ts

```diff
@@ -464,4 +464,17 @@ test('given a prefix, we can import two crds with the same group id', async () =
   });
 
 
-});
\ No newline at end of file
+});
+
+test('can import multiple versions from the same crd', async () => {
+  const fixture = path.join(fixtures, 'multi_version_tenant_crd.yaml');
+  const importer = await ImportCustomResourceDefinition.fromSpec({ source: fixture });
+  await importer.import({ targetLanguage: Language.TYPESCRIPT, outdir: '/tmp/test-tenant' });
+});
+
+
+// test('crds-dev', async () => {
```

Comment:
I see these are commented, can these be removed?

Reply:
Good catch. Removed 👍